scripts/python_tests/README.md

```diff
@@ -0,0 +1,32 @@
+# Nvim_communicator Python Package
+
+This project is made for testing neovim plugins, specifically focussing on Nvim-treesitter-textobjects for now.
+
+It uses pynvim to communicate with neovim, executing lua scripts or input keys and then return the results.
+
+
+## Installation
+
+```bash
+pip install -e .						# requirements for running tools
```

Comment:
this fails for me due to a missing `LICENSE` file
```
❯ pip install -e .
Defaulting to user installation because normal site-packages is not writeable
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Obtaining file:///home/stephan/.local/share/nvim/site/pack/packer/start/nvim-treesitter-textobjects/scripts/python_tests
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [5 lines of output]
      running dist_info
      creating /tmp/pip-modern-metadata-t__wtbbw/nvim_communicator.egg-info
      writing manifest file '/tmp/pip-modern-metadata-t__wtbbw/nvim_communicator.egg-info/SOURCES.txt'
      writing manifest file '/tmp/pip-modern-metadata-t__wtbbw/nvim_communicator.egg-info/SOURCES.txt'
      error: [Errno 2] No such file or directory: 'LICENSE'
      [end of output]
```

Reply:
I just fixed this issue. Can you retry?